### PR TITLE
Javascript TIC-TAC-TOE

### PR DESCRIPTION
**Pull request by Torres-ssf**
Finish the javascript tic-tac-toe assigment

### CODE REVIEW

File: src/game.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+        const positions = winningPositions(boxes);
+        winningCases(positions, { turn,
+            currentPlayer,
+            gameOver, icon,
+            iconTwo,
+            winningMessage });
+
+        turn -= 1;
+        drawGame(turn,GameOver, gameOver);
+        console.log(turn)
```
Comment (JamesOkunlade) [inline src/game.js]: `console.log` affects the performance of an app. Please remove it.

File: src/animation.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
+const removeAnimation = () => {
+    const container = document.querySelector('.container');
+    container.addEventListener('animationend', (event) => {
+        console.log('event added')
```
Comment (JamesOkunlade) [inline src/animation.js]: Remove `console.log`

**Comment by Torres-ssf** [comment]
> ### Hi, devs!! Nice job with this project  The project looks good, however, there are some important fixes you'll need to do 
> * This project follows the lesson about Factory Functions and The Module Pattern so the main goal is to put into practice those concepts along with the main concepts of Object-Oriented Programming. Particularly, classes/objects and how to access their methods and attributes. Refactor your code to use `factories` and `modules` instead. You should revisit [this](https://www.theodinproject.com/courses/javascript/lessons/factory-functions-and-the-module-pattern) to get more clarity about it.
> * Some linting checks are failing, check the details and fix the errors.
> * It's not a very good practice to be using the browser `alert`. Create a simple `UI` for taking player names instead. You can use CSS to hide it back after the names have been submitted.
> * As it is, I can still play the game without putting player names. The players just get assigned `null` as the name. Write a validation code that ensures player names must be given before proceeding. And also, don't use alert to warn the players instead simply display the error message on the UI.
> * `console.log` affects the performance of an app, please remove them from your code.
> 
> **Fix those and request for re-review.**
> 
> **Status: `Changes required`**
> 
> Happy coding!

Thanks for your review and for your suggestions @JamesOkunlade :+1:

**Comment by Ispirett** [comment]
> ### Nice improvements!  But not yet there though.
> Hi guys, I understand how much you'll like to move on to the next project and I promise it's not far. Best practices need to be followed though.
> 
> * You shouldn't use the alert box to take player name inputs, instead, create a form for it that can be hidden once the names have been inputted.
> * As it is now, I still can play the game without players input because I can choose to can the alert box when it comes up. Fixing point one will prevent this.
> 
> **Kindly implement the HTML form for player name inputs and request for re-review.**
> 
> **Status: `Changes required`**
> 
> **Happy coding! sunglasses**

Hey, @JamesOkunlade thanks for the review. but you the change you are asking for is not a required requirement and should not be treated as such. This should be **Optional** change. We are not using` alerts` but` prompts`, its a design choice to allow users to not enter names and receive a generic name. example **player 1** and **player 2**

**Pull request merged**